sz/Traffic/src/traffic/relogo/UserPatch.groovy

```diff
@@ -45,7 +44,28 @@ class UserPatch extends ReLogoPatch{
 	}
 
 	def colorTrafficLights(int x, int y) {
-		setPcolor(green())
+		if (Math.abs(x) > laneWidth && Math.abs(y) > laneWidth) {
+			setPcolor(green())
+		} else if ((x == 0 || y == 0) && (x + y).intdiv(2) % 2 == 0) {
+			setPcolor(white())
+		} else {
+			setPcolor(gray() - 3)
+		}
+	}
+	
+	def changeLights(boolean areTrafficLightsGreenHorizontally ) {
+		def x = getPxcor()
+		def y = getPycor()
+		
+		if (x == laneWidth + 1 && y == laneWidth) { // Right
+			areTrafficLightsGreenHorizontally ? setPcolor(lime()) : setPcolor(red())
+		} else if ((x == (laneWidth + 1) * -1) && (y == laneWidth * -1)) { // Left
```

Comment:
How about writing `-expression` instead of `expression * -1`?

Reply:
Definitely :+1: